script.js

```diff
@@ -1,4 +1,89 @@
+//two game modes
+var gameModeRollDice = "Game mode roll dice";
+var gameModeDiceSequence = "Game mode dice sequence";
```

Comment:
There is something know as [SCREAM_CASE constants ](https://www.freecodecamp.org/news/when-to-capitalize-your-javascript-constants-4fabc0a4a4c4/)

I can touch more on this as the days go by, but it's simply another way to name variables that remain unchanged through out the program eg.
```
var ROLL_DICE_MODE = "Game mode roll dice";
```